Solaris specific fix for Bug 15380373:
SUNBT6527729 Change xhost(1) man page to help with Secure by Default transition
Contributed to OpenSolaris by Richard Lowe <richlowe@richlowe.net>

diff --git a/man/xhost.man b/man/xhost.man
index aac0a0a..cb2f22d 100644
--- a/man/xhost.man
+++ b/man/xhost.man
@@ -181,6 +183,9 @@ the server-interpreted hostname type address.  If somehow you change a
 host's network address while the server is still running, and you are
 using a network-address based form of authentication, \fIxhost\fP must
 be used to add the new address and/or remove the old address.
+.SH WARNINGS
+Oracle Solaris is secure by default.  See Xserver(1) which details how to
+re-enable remote connections to the X server as in previous releases.
 .SH AUTHORS
 Bob Scheifler, MIT Laboratory for Computer Science,
 .br
